### DISCUSSION TITLE
Python Version

### DISCUSSION THREAD

**Discussion by tayler6000** [eigentrustnetwork/ETN#2] [Polls]
Personally, I would prefer at least 3.9 as it has a lot of nice typing updates, it is also the version that Stampy runs on.

**Comment by plexish** [comment]
I defer, @AugustusCeasar, thoughts?

**Comment by AugustusCeasar** [comment]
is there any reason to not go with latest released?

**Reply by tayler6000** [reply]
Python 3.10 is not available on Ubuntu by default, but I'll see if I can build it from source.

**Comment by tayler6000** [comment]
Ok, Python3.10 does exist on Ubuntu 22.04 LTS so we will be using Python 3.10 for this project!
